x-pack/plugins/fleet/server/services/epm/elasticsearch/template/default_settings.ts

```diff
@@ -63,6 +64,19 @@ export function buildDefaultSettings({
 
   const isILMPolicyDisabled = appContextService.getConfig()?.internal?.disableILMPolicies ?? false;
 
+  // TODO populate old defaults when stack templates are disabled?
```

Comment:
With https://github.com/elastic/elasticsearch/pull/99975, disabling stack templates doesn't disable installation of the component templates that are using the new naming convention, such as `metrics@settings`.